content/en/docs/quickstart-sign-non-image-artifact.md

```diff
@@ -0,0 +1,238 @@
+```
+
+Confirm the SBOM is attached to the container image. For example:
+
+```console
+localhost:5000/net-monitor@sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a
+└── example/sbom
+    └── sha256:6cbf7cc5ffa82b030b57ff820d49a86c143d8c6ac483b8e5eead81be8b223fc4
+```
+
+Use an environment variable to store the digest value of the SBOM. This environment variable is used in later steps. For example:
+
+```console
+SBOM=localhost:5000/net-monitor@sha256:6cbf7cc5ffa82b030b57ff820d49a86c143d8c6ac483b8e5eead81be8b223fc4
+```
+
+## List the signatures associated with the SBOM
+
+Use `notation ls` to show any signatures associated with the SBOM you attached in the previous section.
+
+```console
+notation ls $SBOM
+```
+
+Confirm there are no signatures shown in the output.
+
+## Generate a test key and self-signed certificate
+
+Use `notation cert generate-test` to generate a test RSA key for signing artifacts, and a self-signed X.509 test certificate for verifying artifacts.
+
+{{% alert title="Warning" color="warning" %}}
+A self-signed certificate should be used for testing or development purposes only.
+{{% /alert %}}
+
+The following command generates a test key and a self-signed X.509 certificate. With the `--default` flag, the test key is set as a default signing key. The self-signed X.509 certificate is added to a named trust store `wabbit-networks.io` of type `ca`.
+
+```console
+notation cert generate-test --default "wabbit-networks.io"
+```
+
+{{% alert title="Note" color="note" %}}
+At this time, test key and self-signed certificate files created using `notation cert generate-test` can't be removed using only `notation key delete` and `notation cert delete`. For more details on fully removing the test key and self-signed certificate files, see [Remove the test key and self-signed certificate]({{< ref "/docs/installation/uninstall#remove-the-test-key-and-self-signed-certificate" >}}).
+{{% /alert %}}
+
+Use `notation key ls` to confirm the signing key is correctly configured. Key name with a `*` prefix is the default key.
+
+```console
+notation key ls
+```
+
+Use `notation cert ls` to confirm the certificate is stored in the trust store.
+
+```console
+notation cert ls
+```
+
+## Sign the SBOM
+
+Use `notation sign` to sign the SBOM.
+
+```console
+notation sign $SBOM
+```
+
+By default, the signature format is `JWS`. Use `--signature-format` to use [COSE](https://datatracker.ietf.org/doc/html/rfc8152/) signature format.
+
+```console
+notation sign --signature-format cose $SBOM
+```
+
+The generated signature is pushed to the registry and the digest of the SBOM is returned.
+
+Use `notation ls` to show the signature associated with the SBOM.
+
+```console
+notation ls $SBOM
+```
+
+Confirm there is one signature, for example:
+
+```output
+$ notation ls $SBOM
+localhost:5001/net-monitor@sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a
+└── application/vnd.cncf.notary.v2.signature
```

Comment:
```suggestion
└── application/vnd.cncf.notary.signature
```

Reply:
There is no signature with `application/vnd.cncf.notary.v2.signature` type now.

Reply:
Fixed